lib/sinatra/base.rb

```diff
@@ -1936,7 +1925,7 @@ class #{self.class}
             </style>
           </head>
           <body>
-            <h2>Sinatra doesn&rsquo;t know this ditty.</h2>
+            <h2>Sinatra doesn’t know this ditty.</h2>
```

Comment:
If we're going to change this at all I'd rather replace the quote mark with a proper apostrophe (`&apos;`). Personal preference, totally not a blocker. =^)